.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+_Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change._
+
+
+## Type of Change
+
+Please delete options that are not relevant.
+
+* Bug fix (non-breaking change which fixes an issue)
+* New feature (non-breaking change which adds functionality)
+* Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
Should we encourage introducing breaking change? I think this should be something very rare to do, so not sure about adding this type.

Reply:
good point :)